### ISSUE TITLE
Missing (Voiceover) sounds for DomoAnimate Domo Theme

### ISSUE THREAD

**Issue by johnnybob78** [wrapper-offline/wrapper-offline#145]
Would it be okay if you can readd all the voice sounds from the voiceover folder of the Domo Theme from DomoAnimate to Sounds here? You can either add them all in the Sounds folder or just add the voiceover folder in Sounds exclusively to the theme?
